manifests/init.pp

```diff
@@ -214,25 +236,21 @@
       $_arg_workers = ''
     }
 
-    $cmd = '/usr/bin/createrepo'
+    $cmd = "/usr/bin/${createrepo_package}"
     $_arg_cachedir = "--cachedir ${repo_cache_dir}"
     $arg = "${_arg_cachedir}${_arg_changelog}${_arg_checksum}${_arg_groupfile}${_arg_workers}"
     $cron_output_suppression = "${_stdout_suppress}${_stderr_suppress}"
     $createrepo_create = "${cmd} ${arg} --database ${repository_dir}"
     $createrepo_update = "${cmd} ${arg} --update ${repository_dir}"
-    $repomanage_cleanup = "rm $(/usr/bin/repomanage --keep=${cleanup_keep} --old ${repository_dir})"
+    $repomanage_cleanup = "/usr/bin/repomanage --keep=${cleanup_keep} --old ${repository_dir} | xargs -r rm"
```

Comment:
This wasn't documented in the PR summary. In this case `xargs` is not using a full path which contradicts the rest of the module. Is there a reason for this change?

Reply:
The reason for this change is to avoid an error message whenever repomanage returns an empty list. This possibly also helps if it returns a lot of old packages. I will put a full path here.